src/fragments/lib/in-app-messaging/js/overview.mdx

```diff
@@ -0,0 +1,23 @@
+<Callout >
```

Comment:
Should this have an `informational` prop?

Reply:
I copied it from the Push Notifications category. By default severity is `informational` so, it will render the same style.

Reply:
We have added the `informational` prop to the other callouts, so we should either add one here or remove the others

Reply:
added the `informational` to the overview page.